pkg/data/helpers.go

```diff
@@ -37,3 +45,81 @@ func getFiltersFromStruct(i interface{}, keyName *string) (*expression.KeyCondit
 	}
 	return kb, cb
 }
+
+func putItem(input interface{}, data interface{}, tableName string, expr *expression.Expression) error {
```

Comment:
Another option here is to 
`func putItem(input *dynamodb.PutItemInput) error`

Then WriteAccount can just be building that input and passing it along.

Reply:
Yes, But you have data layer struct type Account struct {} that needs to be made generic to be called DataLayer struct {} and then we can make it completely generic by 

`func putItem(input *dynamodb.PutItemInput, dataInterface *DataLayer) error